extensions/amp-form/0.1/test/test-amp-form.js

```diff
@@ -107,6 +107,17 @@ describe('amp-form', () => {
     expect(() => new AmpForm(form)).to.not.throw;
   });
 
+  it('should assert none of the inputs named __amp_source_origin', () => {
+    const form = getForm(document, true, false);
+    const illegalInput = document.createElement('input');
```

Comment:
FYI, you should be able to use the util method in `dom.js`
`createElementWithAttributes`.